_posts/2022-08-09-Emmanuel-Cortes.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Molecular Simulations: Cutoffs and Periodic Boundaries"
+author: Emmanuel Cortes
+---
+## Cutoffs
+Cutoff distances are utilized extensively by molecular simulations to reduce the amount of calculations required. Cutoffs are based on the heuristic that particles afar from one another produce negligible forces. However, given cutoffs are a heuristic device, the method might ignore anomalies or potentially significant interactions.
+
+
+If we have a system of 10 particles, then there will be 45 pairwise particle-particle interactions to consider. We know this since the number of interactions for a N particle system is `N choose 2`. Thus, for 100 particle system, there will have 4950 interactions. And for 1000 interactions, there will be 499500 interactions. As you can see, the number of interactions increases quite drastically and thus the need for cutoffs in simulations.
+
+| # particles | # pairwise interactions |
+|---|---|
+|10|45|
+|100|4950|
+|1000|499500|
+
+Given a system with a density of 1, then in the table we can see the number of particles in spheres with varying radii.
+
+| radius (unit) | # particles in sphere |
```

Comment:
must be nice to include some sort of explanation like with the previous question.